apps/playground/app/sink/page.tsx

```diff
@@ -232,7 +232,7 @@ export default function Sink() {
                               </Button>
                             </Tooltip>
 
-                            <Tooltip content="The goal of typography is to relate font size, line height, and line width in a proportional way that maximizes beauty and makes reading easier and more pleasant.">
+                            <Tooltip content="The goal of typography is to relate font size, line height, and line width in a proportional way that maximizes beauty and makes reading easier and more pleasant.">
```

Comment:
There were a bunch of weird, inconsistent whitespace characters in this file so I just replaced them all.